### PR TITLE
Fix for tests

### PR DESCRIPTION
**Pull request by PoLaKoSz**
This PR reduce the failing tests count close to 0.

### CODE REVIEW

**Comment by PoLaKoSz** [comment]
ping @projectgoav 
If You merge this:
❌ 0 Tests Failed
⚠️ 5 Tests Skipped
✔️ 188 Tests Passed

File: E.Deezer/Api/Album.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public interface IAlbum : IObjectWithImage
         string Link { get; }
         string Title { get; }
         string Label { get; }
-        uint GenreId { get; }
+        int GenreId { get; }
```
Comment (projectgoav) [inline E.Deezer/Api/Album.cs:22]: Think we should change all `GenreId` properties to be `int` to keep everything consistant

File: E.Deezer/Api/GenreWithRadios.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace E.Deezer.Api
+{
+    public class GenreWithRadios : IGenreWithRadios
```
Comment (projectgoav) [inline E.Deezer/Api/GenreWithRadios.cs:8]: Not sure about this class name. If this is just a response class should probably have this in it's name. 

Also! Class should be private

File: E.Deezer/Endpoint/BrowseEndpoint.cs [hunk 3]
```diff
@@ -69,10 +69,12 @@ public async Task<IAlbum> GetAlbumById(uint albumId)
             };
 
             // Will throw if Deezer error
-            var response = await iClient.GetDeezerObject<AlbumObjectResponse>("album/{id}", p)
+            var response = await iClient.GetPlain<Album>("album/{id}", p)
```
Comment (projectgoav) [inline E.Deezer/Endpoint/BrowseEndpoint.cs]: Not sure if this is correct. I'll need to check this out....

**Review by projectgoav — CHANGES_REQUESTED** [review]

File: E.Deezer/Api/Album.cs (hunk 1 above)
Comment (PoLaKoSz) [reply E.Deezer/Api/Album.cs:22]: I ran a quick search on the Entire Solution looking for `uint GenreId` but found nothing.

`Find all "uint GenreId", Subfolders, Find Results 1, Entire Solution, ""
  Matching lines: 0    Matching files: 0    Total files searched: 117`

File: E.Deezer/Api/GenreWithRadios.cs (hunk 2 above)
Comment (PoLaKoSz) [reply E.Deezer/Api/GenreWithRadios.cs:8]: Yes, i agree with You this is not the best name but i wasn't able to come up with a better one. Do You have any idea how it should be called? This is a response class, like the `Artist`, `Genre`, etc.

File: E.Deezer/Endpoint/BrowseEndpoint.cs (hunk 3 above)
Comment (PoLaKoSz) [reply E.Deezer/Endpoint/BrowseEndpoint.cs]: Yes, You were wright this was a mistake from me. I pushed a new commit which revert it to the original version and added a few tests just to make sure it won't happen again (**78.05%** code coverage).

**Comment by PoLaKoSz** [comment]
The `GenreWithRadios` class has unique properties (only `ID`, `Title` and `IEnumerable<T>`) which doesn't occur in any other server response so i don't have any idea for a better name.

**Review by projectgoav — APPROVED** [review]

**Pull request merged**